tools/osbuilder/igvm-builder/igvm_builder.sh

```diff
@@ -61,6 +64,7 @@ echo "-- OUT_DIR -> $OUT_DIR"
 echo "-- SVN -> $SVN"
 echo "-- DISTRO -> $DISTRO"
 echo "-- MODE -> $MODE"
+echo "-- DEBUG -> $DEBUG"
```

Comment:
now that I look at this, we should at least add a comment saying that SVN and DEBUG are being used by build_igvm_files